charter.md

```diff
@@ -37,12 +37,17 @@ decryption of media payloads where required.
 
 The working group will define MoQ so that the media publication protocol 
 can leverage coodinating relays, caches, or replication points wherever applicable 
-to improve the delivery performance. Media will be encrypted, possibly 
-end-to-end encrypted for certain usecases. The keying mechanisms for media 
-confidentiality is however outside the scope of this working group. Even when media is
-end-to-end encrypted, the relays can access metadata needed for caching
-(such as timestamp), making media forwarding decisions (such as drop or
-delay under congestion) and so on.
+to improve the delivery performance.
+
+Media will be encrypted at the transport layer using the standard QUIC
+mechanisms.  Media content may be end-to-end encrypted in certain use cases,
+where the "end-to-end" keys are avialable to media sources and consumers, but
+not relays.  Even when media is end-to-end encrypted, the relays can access
+metadata needed for caching (such as timestamp), making media forwarding
+decisions (such as drop or delay under congestion) and so on.  Such media
```

Comment:
```suggestion
decisions (such as drop or delay under congestion), and so on.  Such media
```

"and so on" is a bit weak, but I don't have a suggestion here.  How we decide what is OK for a relay to see is important, almost as important as what we decide is OK.

Reply:
I didn't change that text :)  I'm a little split -- on the one hand, the choice of metadata is clearly important for privacy, but on the other hand, there's not much we can do to constrain things.

Reply:
i think we need to clarify that by saying 
for relays to make drop/forward/store decisions and effectively react to congestion feedback from the underlying QuiC Transport.

Reply:
@martinthomson would that work ?